### PR TITLE
Added graph parser and test project

### PR DESCRIPTION
**Pull request by nikolai-kosolapov**

### CODE REVIEW

File: Graphs/Parsers/GraphParser.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            var nodeValuesStr = str[0];
+            var nodes = GetCollectionOfNodes(nodeValuesStr).ToArray();
+
+            // create relationships
+            CreateRelationShips(str, nodes);
+
+            return nodes;
+        }
+
+        private static void CreateRelationShips(string[] str, Node<string>[] nodes)
```
Comment (nikolai-kosolapov) [inline Graphs/Parsers/GraphParser.cs:27]: Fix method's name

**Review by nikolai-kosolapov — COMMENTED** [review]
Fix method's name

**Review by nikolai-kosolapov — COMMENTED** [review]
Add a test for this exception

File: Graphs/Parsers/GraphParser.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    using Graphs.Models;
+
+    public class GraphParser : IGraphParser
+    {
+        public IList<Node<string>> GetStringGraphByString(string[] str)
+        {
+            // get collection of nodes
+            if (str.Length <= 1)
+            {
+                throw new Exception("Array should have more than one string");
```
Comment (nikolai-kosolapov) [inline Graphs/Parsers/GraphParser.cs:15]: Add a test for this exception

**Review by cate-ok — APPROVED** [review]

**Pull request merged**